javatests/build/bazel/tests/integration/BUILD

```diff
@@ -11,6 +11,9 @@ bazel_java_integration_test(
         "//java/build/bazel/tests/integration",
         "@com_google_truth//jar",
     ],
+    # We had to disable this test on Windows because it will fail due to running Bazel
+    # from a path longer than 256 charactors.
+    tags = ["no_windows"],
```

Comment:
Can you file an issue for this?

Reply:
Better than that, I figured out a workaround ;)